### ISSUE TITLE
Moodle phpunit tests fail with boost_union installed on 4.1 due to customfiletypes

### ISSUE THREAD

**Issue by radzisze** [moodle-an-hochschulen/moodle-theme_boost_union#305]
Moodle 4.1.3
Boost union v4.1-r7

When running Moodle's phpunit tests with boost_union installed I noticed this Moodle core test started failing: 
```
core\filetypes_test::test_cleanup
Failed asserting that object of class "stdClass" does not have attribute "customfiletypes".
/home/travis/build/isteclass/eclass/lib/tests/filetypes_test.php:217
/home/travis/build/isteclass/eclass/lib/phpunit/classes/advanced_testcase.php:80
phpvfscomposer:///home/travis/build/isteclass/eclass/vendor/phpunit/phpunit/phpunit:97
```
It appears that Moodle assumes that customfiletypes should be empty at the start of that test, and customfiletypes are added in function theme_boost_union_register_webfonts_filetypes() of locallib.php of boost_union. There is a nice check at the start of this function already to see if customfiletypes are set in config.php that just returns false. Perhaps adding a check for if phpunit is running or something could be added there (" || PHPUNIT_TEST" maybe?) or in that function to prevent adding customfiletypes during phpunit test runs as well?

**Comment by lucaboesch** [comment]
Dear @radzisze I crafted https://github.com/lucaboesch/moodle-theme_boost_union/tree/phpunitcustomfiletypes based on your proposal. Could you check out if that works for you?

**Comment by abias** [comment]
This was solved by #306
